docs/api-and-integration/metadata/index.md

```diff
@@ -0,0 +1,116 @@
+
+As a second example, imagine you have two environments in a lifecycle, but you don't always deploy to one of them. Let's use staging and a performance test environment as an example. Maybe you periodically deploy to the performance environment to check for regressions but not every time you deploy to staging. The work-items for the performance environment would be the same as staging plus the additional work-items for the releases in between.
+
+## Deploy a Release Step
+
+The Octopus [deploy release step](https://g.octopushq.com/DeployReleaseStep) adds an interesting dimension to metadata accumulation. When you use this step Octopus treats each "child project" like it is a package. Following from this reasoning, it also treats the child project's release as a potential source for metadata.
+
+On the project settings you can select a deploy release step and use it the same as any other package from the internal feed. When creating the releases in the "parent project" Octopus accumulates the release notes and metadata from the child projects just as it would for packages.
+
+## Deployment Variables and the Email Step
+
+During a deployment there are variables available for both the new release notes values and the work-items.
+
+The release notes variable is `Octopus.Deployment.Changes` and contains the release notes and work-items in JSON format. The structure is a JSON array of `ReleaseChange` objects matching the following C# class:
+
+```csharp
+public class ReleaseChanges
+{
+  public string Version { get; set; }
+  public string ReleaseNotes { get; set; }
+  public WorkItemLink[] WorkItems { get; set; }
+}
+
+public class WorkItemLink
+{
+    public string Id { get; set; }
+    public string LinkUrl { get; set; }
+    public string LinkText { get; set; }
+}
+```
+
+There is an entry per release and it includes the release notes and the metadata for each of the packages in that release.
+
+The following example uses these variables to generate the HTML body for the Octopus email step.
+
+```html
+Here are the notes:<br/>
+#{each change in Octopus.Deployment.Changes}
+<h2>#{change.Version}</h2>
+#{change.ReleaseNotes}</br>
+#{each workItem in change.WorkItems}
+  #{if workItem.LinkUrl}
+      <a href="#{workItem.LinkUrl}">#{workItem.LinkText}</a>
+  #{/if}
+  #{unless workItem.LinkUrl}
+      #{workItem.LinkText}</br>
+  #{/unless}
+#{/each}
+#{/each}
+```
+
+Note that the if/unless is included for illustration purposes. The links can end up without a LinkUrl if the related extension (e.g. Jira) is disabled. If you know it's going to be enabled then just use the anchor.
+
+## Issue Trackers
+
+If you are using an issue tracker like Jira, Octopus can also be integrated with it to provide information about the progress of work-items. For more details see
```

Comment:
```suggestion
If you are using an issue tracker like Jira, Octopus can also be configured to provide information about the progress of work-items. For more details see
```


Actually, I'm not 100% sure what this sentence means. It sit pushing info to Jira? Or is it showing info from Jira in Octopus?